server/org.eclipse.emfcloud.ecore.glsp/src/main/java/org/eclipse/emfcloud/ecore/glsp/gmodel/GModelFactory.java

```diff
@@ -98,7 +99,7 @@ public GGraph create(EPackage ePackage) {
 		// create reference edges
 		eClass.getEReferences().stream().map(this::create).filter(Objects::nonNull).forEach(children::add);
 		// create inheritance edges
-		eClass.getESuperTypes().stream().map(s -> create(eClass, s)).forEach(children::add);
+		eClass.getEGenericSuperTypes().stream().map(this::create).forEach(children::add);
```

Comment:
Generalization Edges represent an "eSuperType" (Of type EClass). EGeneric(Super)Type is a different concept, that is used to represent "Java Generic Types" (Which we currently don't support in Ecore GLSP). I don't think this change is correct

Reply:
Ah thanks for pointing that out! As we discussed, I updated my changes accordingly.